Error: Current page must be indicated using `aria-current="page"`
Error: Nothing in the requested component required `aria-expanded` be used
Error: Nothing in the requested component required `aria-controls` be used
Error: Styles were not requested
diff --git a/output/Basic Navigation with Current Page/4/717a8cbe-1cda-4980-a76a-b802932267a3.html b/output/Basic Navigation with Current Page/4/717a8cbe-1cda-4980-a76a-b802932267a3.html
index 1ae6c94..8c657cb 100644
--- a/output/Basic Navigation with Current Page/4/717a8cbe-1cda-4980-a76a-b802932267a3.html	
+++ b/output/Basic Navigation with Current Page/4/717a8cbe-1cda-4980-a76a-b802932267a3.html	
@@ -2,7 +2,7 @@
   <ul>
     <li><a href="/">Home</a></li>
     <li>
-      <a href="/products" aria-expanded="true" aria-controls="subnav-products">Products</a>
+      <a href="/products" aria-current="page">Products</a>
       <ul id="subnav-products">
         <li><a href="/products/frobs">Frobs</a></li>
         <li><a href="/products/thrips">Thrips</a></li>
@@ -10,9 +10,4 @@
     </li>
     <li><a href="/services">Services</a></li>
   </ul>
-</nav>
-<style>
-  nav ul ul {
-    display: block;
-  }
-</style>
\ No newline at end of file
+</nav>
\ No newline at end of file
